include/linux/soundwire/sdw.h

```diff
@@ -961,6 +980,8 @@ struct sdw_stream_runtime {
  * @defer_msg: Defer message
  * @params: Current bus parameters
  * @stream_refcount: number of streams currently using this bus
+ * @btp_stream_refcount: number of BTP streams currently using this bus (should
+ * be zero or one, multiple streams per link is not supported).
```

Comment:
It can be a bool, right? bpt_stream_active,

Reply:
It could be, but I prefer refcount, just like `stream_refcount`. BPT is still a stream.